docker-hub/publish/insights-analytics.md

```diff
@@ -141,3 +131,37 @@ target="_blank" rel="noopener" class="_"}.
 | GET            | digest    | GET by different digest                                         | Pull by digest   | Image is multi-arch                                                                                            | The second GET by digest must be different from the first                                                                                                                                                                                                                                      |
 | HEAD           | digest    | GET by same digest                                              | Pull by digest   | Image is single arch and/or image is multi-arch but some part of the image already exists on the local machine |
 | HEAD           | digest    | GET by same digest, then a second GET by different digest       | Pull by Digest   | Image is multi-arch                                                                                            |
+
+## Changes in data over time
+
+The insights and analytics service is continuously improved to increase the
+value it brings to publishers. Some changes might include adding new data
+points, or improving existing data to make it more useful.
+
+When there is a change in the dataset provided by the service, such a change
+doesn't get retroactively applied. As new data points get added, they're
+available from the point of introduction and going forward.
+
+Refer to the tables in the [Data points](#data-points) section to see from which
+date a given data point is available.
+
+## Privacy
+
+This section contains information about privacy-protecting measures that ensures
+consumers of content on Docker Hub remain completely anonymous.
+
+> **Important**
+>
+> Docker never shares any Personally Identifiable Information (PII) as part of
+> analytics data. {: .important }
+
+The summary dataset includes Unique IP address count. This data point only
+includes the number of distinct unique IP addresses that request an image.
+Individual IP addresses are never shared.
+
+The raw dataset includes user IP domains as a data point. That's the domain name
+of the company associated with an IP address that pulled an image (for example,
+`docker.com`). This data point is only included where the IP type is `business`,
```

Comment:
In the end, we went with always including this data point, but it will only return the end-consumer when it is of type `business`. In other cases it will return the domain of the ISP or hosting provider. CC @Bkblodget

Reply:
OK, updated the desc... PTAL!

Reply:
Looks good to me! Worth getting Britney to take a look and then we are probably good.